concepticondata/conceptlists/Seifart-2015-414.tsv

```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-340	340	basket	1539	BASKET	456
+Seifart-2015-414-341	341	foam	1768	FOAM	458
+Seifart-2015-414-342	342	middle	1093	MIDDLE	461
+Seifart-2015-414-343	343	thorn	124	THORN	462
+Seifart-2015-414-344	344	tall	711	TALL	463
+Seifart-2015-414-345	345	palm oenocarpus bataua			464
+Seifart-2015-414-346	346	medicine	1372	MEDICINE	465
+Seifart-2015-414-347	347	later on			467
+Seifart-2015-414-348	348	a lot			468
+Seifart-2015-414-349	349	behind	1686	BEHIND	469
+Seifart-2015-414-350	350	full	1429	FULL	471
+Seifart-2015-414-351	351	peanuts	2364	PEANUT	474
+Seifart-2015-414-352	352	yard,court (of maloca)	736	YARD	475
+Seifart-2015-414-353	353	crested oropendola			476
+Seifart-2015-414-354	354	manioc starch drink			477
+Seifart-2015-414-355	355	arrive	1387	ARRIVE	478
+Seifart-2015-414-356	356	oil	1484	OIL (ORGANIC SUBSTANCE)	479
+Seifart-2015-414-357	357	sifter	3353	SIEVE (TOOL)	482
+Seifart-2015-414-358	358	spicy	544	SPICY	484
+Seifart-2015-414-359	359	dancing festival			489
+Seifart-2015-414-360	360	manioc squeezer			491
+Seifart-2015-414-361	361	mirror	1171	MIRROR	492
+Seifart-2015-414-362	362	son	1620	SON	494
+Seifart-2015-414-363	363	we two	2634	WE TWO	495
+Seifart-2015-414-364	364	you two	2111	YOU TWO	496
+Seifart-2015-414-365	365	they two (m.)	2635	THEY TWO	497
+Seifart-2015-414-366	366	they two (f.)	2635	THEY TWO	498
+Seifart-2015-414-367	367	her…			499
+Seifart-2015-414-368	368	eat meat			500
+Seifart-2015-414-369	369	weave	133	WEAVE	501
+Seifart-2015-414-370	370	lick	319	LICK	502
+Seifart-2015-414-371	371	healing blow			503
+Seifart-2015-414-372	372	downriver			504
+Seifart-2015-414-373	373	wake up	1602	WAKE UP	506
+Seifart-2015-414-374	374	fireplace	185	FIREPLACE	507
+Seifart-2015-414-375	375	sieve	3353	SIEVE (TOOL)	508
+Seifart-2015-414-376	376	dancing beam			509
+Seifart-2015-414-377	377	caimito fruit (pouteria caimito)			510
+Seifart-2015-414-378	378	palm socratea exorrhiza			511
+Seifart-2015-414-379	379	leaves that produce black color			512
+Seifart-2015-414-380	380	peach palm (bactris gasipaes)	3919	PEACH PALM	513
+Seifart-2015-414-381	381	cecropia	3931	CECROPIA	514
+Seifart-2015-414-382	382	macambo fruit (theobroma bicolor)			515
+Seifart-2015-414-383	383	inga edulis	3981	GUAVA	518
+Seifart-2015-414-384	384	ceramic saucepan			519
+Seifart-2015-414-385	385	uvilla fruit (pourouma cecropiaefolia)			520
+Seifart-2015-414-386	386	rifle	3881	PISTOL	521
+Seifart-2015-414-387	387	palm astrocaryum chambira			522
+Seifart-2015-414-388	388	umarí fruit (poraqueiba sericea)			523
+Seifart-2015-414-389	389	edible ant			525
+Seifart-2015-414-390	390	avocado (persea americana)	3725	AVOCADO	526
+Seifart-2015-414-391	391	edible larvae			527
+Seifart-2015-414-392	392	grater			529
+Seifart-2015-414-393	393	miriti palm (mauritia flexuosa)	3811	MIRITI PALM	530
+Seifart-2015-414-394	394	pillars (of roundhouse)			532
+Seifart-2015-414-395	395	lepidocaryum tenue palm (for thatching roofs)			533
+Seifart-2015-414-396	396	eat fruit			534
```

Comment:
Could someone do a study on EAT colexifications with different kinds of food? Or is there already one? I've never seen this distinction in a concept list before.

Reply:
There is but just [Kaufman-1973-1028-775](https://concepticon.clld.org/values/Kaufman-1973-1028-775).